### PR TITLE
JAVA-3243: Deprecate DBCursor.explain

### PR DESCRIPTION
**Pull request by jstewart-mongo**

### CODE REVIEW

File: driver-legacy/src/main/com/mongodb/DBCursor.java [hunk 1]
```diff
@@ -335,7 +335,10 @@ public DBCursor maxTime(final long maxTime, final TimeUnit timeUnit) {
      * @return a {@code DBObject} containing the explain output for this DBCursor's query
      * @throws MongoException if the operation failed
      * @mongodb.driver.manual reference/explain Explain Output
+     * @deprecated This method is no longer part of our CRUD API. In 4.x this method will only support MongoDB 3.2 and up (where the
```
Comment (jyemin) [inline driver-legacy/src/main/com/mongodb/DBCursor.java]: No need to mentions 4.x if the deprecation is happening in 4.0

Comment (jstewart-mongo) [reply driver-legacy/src/main/com/mongodb/DBCursor.java]: I have altered the deprecation comment to not mention 4.x.

**Comment by rozza** [comment]
+1 for deprecating in 3.12

**Comment by jyemin** [comment]
I agree, so let's close this PR and open a new one on top of master.

**Comment by jstewart-mongo** [comment]
Closing this PR to move this to master.

**Pull request closed without merging**